about.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <script
+      src="https://kit.fontawesome.com/eca3ff88d7.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@500;600;700&family=Lato:ital,wght@0,400;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="stylesheet" href="./css/about.css" />
+    <title>Capstone About</title>
+  </head>
+  <body>
+    <header>
+      <nav class="header-nav">
+        <ul class="list header-nav__list">
+          <li class="header-nav__item">
+            <a href=""><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li class="header-nav__item">
+            <a href=""><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li class="header-nav__item"><a href="">English</a></li>
+          <li class="header-nav__item"><a href="">My page</a></li>
+          <li class="header-nav__item"><a href="">My page</a></li>
+        </ul>
+      </nav>
+      <nav class="nav">
+        <a href="../index.html">
+          <svg class="icon brand-icon">
+            <use xlink:href="./images/sprite.svg#remote-work"></use>
+          </svg>
+        </a>
+        <div class="nav__toggler">
+          <span class="toggler-bar first-bar"></span>
+          <span class="toggler-bar second-bar"></span>
+          <span class="toggler-bar third-bar"></span>
+        </div>
+        <ul class="list nav__list collapsible__content">
+          <li class="nav__item"><a href="./index.html">Home</a></li>
+          <li class="nav__item">
+            <a href="about.html">About</a>
+          </li>
+          <li class="nav__item"><a href="#">Programs</a></li>
+          <li class="nav__item active-item"><a href="#">Contact Us</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="about-info">
+      <div class="container">
+        <h1 class="about-heading">JOIN OUR REMOTE EXPERTS CONFERENCE 2022</h1>
+        <p class="about-description">
+          At REMOTE EXPERTS we believe that everyone deserves free access to
+          world-class education. We offer free online courses offers free online
+          education courses . And under each category we provide , there are
+          numerous subjects for almost every remote career.
+        </p>
+        <p class="contact-message">
+          Pleade contact me fot more information about the program
+        </p>
```

Comment:
- [ ] [REQUIRED] I would recommend that you remove the background image obstructing the `p` tag for a better UI. 
### Your Design
![Screenshot from 2022-03-18 14-23-02](https://user-images.githubusercontent.com/58692590/159011475-da78ab14-077e-4786-9c7a-51b1b93dcd73.png)